ckanext/ytp/request/tests/test_plugin.py

```diff
@@ -56,10 +56,11 @@ def test_viewing_actioned_referral(self, mocked_create_flash, mocked_update_flas
             action='show',
             mrequest_id=membership_request['id']            
         )
-        response = app.get(
-            url,
-            extra_environ={'REMOTE_USER': sysadmin['name'].encode('ascii')},
-            status=200
-        )
-
-        # app.get() will raise a "Bad response" error if the page 404s
\ No newline at end of file
+        try:
+            app.get(
+                url,
+                extra_environ={'REMOTE_USER': sysadmin['name'].encode('ascii')},
+                status=200
+            )
+        except BadResponseError:
```

Comment:
That's not going to work. `BadResponseError` is undefined in your code. Travis has caught that as well:
https://travis-ci.org/github/fjelltopp/ckanext-ytp-request/jobs/736091133